Source/WebCore/css/CSSProperties.json

```diff
@@ -1146,6 +1146,25 @@
                 "non-canonical-url": "https://drafts.csswg.org/css-size-adjust/#adjustment-control"
             }
         },
+        "text-autospace": {
+            "inherited": true,
+            "values": [
+                "auto",
+                "no-autospace"
+            ],
+            "codegen-properties": {
+                "settings-flag": "cssTextSpacingEnabled",
+                "converter": "TextAutospace",
+                "parser-function": "consumeTextAutospace",
+                "parser-grammar-unused": "",
```

Comment:
Can you fill in this field? it's helpful to audit missing features of the new parser